CHANGELOG.rst

```diff
@@ -2,6 +2,22 @@
 Change history
 ==============
 
+0.6.0 (2025-10-02)
+------------------
+
+* Upgrade dependencies
+
+    * Django to 5.2.7
+    * open-api-framework to 0.13.0
+    * zgw-consumers to 1.0.0
+    * django-privates to 3.1.1
+    * commonground-api-common to 2.10.0
```

Comment:
I did releases for OAF and commonground-api-common to fix some logging issues, could you upgrade to those new versions?
```suggestion
    * open-api-framework to 0.13.1
    * zgw-consumers to 1.0.0
    * django-privates to 3.1.1
    * commonground-api-common to 2.10.1
```